server/routes/topic_page.py

```diff
@@ -55,5 +72,22 @@ def topic_page(topic_id=None, place_dcid=None):
         place_type=place_type,
         place_name=place_name,
         place_dcid=place_dcid,
-        topic_name=topic_place_config.metadata.topic_name,
-        config=MessageToJson(topic_place_config))
+        topic_id=topic_id,
+        topic_name=topic_place_config.metadata.topic_name or "",
+        config=MessageToJson(topic_place_config),
+        topics_summary=topics_summary)
+
+
+def get_topics_summary():
```

Comment:
we could add this to the flask cache since we'll need it for each topic page render

Reply:
Did you mean add it to the app config or adding the `@cache.memoize`. I added it to the app config because the topics summary will always be the same

Reply:
app config sgtm